core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-gallery/ai-image-prompt-gallery.component.scss

```diff
@@ -0,0 +1,60 @@
+@use "variables" as *;
+
+:host {
+    display: flex;
+    flex-direction: column;
+}
+
+.ai-image-gallery__placeholder {
+    background-color: $color-palette-gray-200;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: $border-radius-md;
+
+    &.s1024x1792 {
+        aspect-ratio: 1/1.25;
+    }
+
+    &.s1792x1024 {
+        aspect-ratio: 1/0.5;
+    }
+
+    &.s1024x1024 {
+        aspect-ratio: 1/1;
+    }
+}
+
+p-galleria {
+    height: 400px;
```

Comment:
I think this style will affect globally

Reply:
Angular will scope it to this component

Reply:
As long as is not wrapped in a `:ng-deep` we are safe